progenitor-impl/tests/test_output.rs

```diff
@@ -26,6 +26,16 @@ where
     }
 }
 
+fn generate_formatted(generator: &mut Generator, spec: &OpenAPI) -> String {
```

Comment:
Let's make the goal to have no changes in the output fixtures

Reply:
There is still one changed fixture - `progenitor-impl/tests/output/nexus-cli.out` - this is because the removed formatting code in cli.rs had different rustfmt settings, specifically it used only `format_strings: Some(true),` which wasnt used in lib.rs.

Reply:
What's it look like if we `format_strings: Some(true)` for everything?

Reply:
IMO it looks great - it does however change the non-cli .out files, so was optional which approach to take.  I'll add it and you can judge for yourself, but IMO it is worth the extra changes.